assets/src/vo/index.js

```diff
@@ -1,15 +1,4 @@
 export { default as Money } from './money';
-export {
-	default as SiteCurrency,
-	Currency,
-} from './currency';
+export { default as SiteCurrency, Currency } from './currency';
 export { default as Label } from './label';
-export {
-	DateTime,
-	Duration,
-	ServerDateTime,
-	DEFAULT_TIMEZONE_STRING,
-	DEFAULT_OFFSET,
-	DEFAULT_VALID_LOCALE,
-	DEFAULT_FORMAT,
-} from './date-time';
+export { DateTime, Duration, ServerDateTime } from './date-time';
```

Comment:
Where did the constants in here get moved to? (`DEFAULT_TIMEZONE_STRING`, `DEFAULT_OFFSET` etc?)

Reply:
These constants were not initially being exported here.
I needed to use them for something and so had added this code only to realize later that they could be obtained elsewhere.

Anyways, you had asked for these exports to be removed during your initial review of this pull.

from: https://github.com/eventespresso/event-espresso-core/pull/1265#discussion_r289378360

> This exports these constants on the @eventespresso/value-objects object. I think if you have need for these constants to be exposed, then it'd be better to expose them as a part of the @eventespresso/helpers export

so this commit merely returns this file back to its original state

Reply:
Oh gosh, I think this is because I looked at a subset of changes in this review instead of _all_ the changes 🤦‍♂ .  My bad, looks good then 👍 .